README.md

```diff
@@ -129,6 +129,31 @@ The script will load the ZKPI data for the specified block, convert it to the
 format required by Keth, run the proof generation process, and save proof
 artifacts to the output directory.
 
+### Running State Transition Function from ZKPI
+
+Execute an Ethereum block stateless in Python. That is: execute and verify the
+Ethereum state transition function in Python (no-proof mode) using a ZK-Prover
+Input (ZKPI) file, use the `stf_from_zkpi.py` script:
+
+```bash
+uv run stf_from_zkpi <BLOCK_NUMBER>
```

Comment:
naming problem, we already have `zkpi_TO_xxx`, please don't introduce `xxx_FROM_yyy`